pulp_container/tests/functional/api/test_push_signatures.py

```diff
@@ -73,9 +73,7 @@ def test_assert_signed_image(
         decrypted = gpg.decrypt(raw_s)
 
         assert decrypted.key_id == keyid
-        assert decrypted.fingerprint == fingerprint
```

Comment:
The fingerprint comes from the bindings. I believe that should be fine. Maybe I'm missing something again.

Reply:
See previous comment, fingerprint and keyid are effectively the same, as one (I believe keyid) is just the last X bits of the other.  Realistically just checking one is the same as checking both as they are both meant to be functionally unique values.